src/grafana_dashboards/discourse.json

```diff
@@ -0,0 +1,2077 @@
+{
```

Comment:
I'll skip reviewing this line by line, is this auto generated somehow?

Reply:
Not auto generated but based on an official template. I've changed the datasource, removed the id and changed the descriptions and links too.